openedx/features/qverse_features/registration/tasks.py

```diff
@@ -0,0 +1,93 @@
+"""
+Qverse registration application celery tasks.
+"""
+from celery.task import task
+from celery.utils.log import get_task_logger
+from django.conf import settings
+from django.contrib.auth.models import User
+from edx_ace import ace
+from edx_ace.recipient import Recipient
+
+from openedx.core.djangoapps.ace_common.template_context import get_base_template_context
+from openedx.core.djangoapps.site_configuration import helpers as configuration_helpers
+from openedx.core.djangoapps.theming.helpers import get_current_site
+from openedx.core.lib.celery.task_utils import emulate_http_request
+from openedx.features.qverse_features.registration.message_types import RegistrationNotification
+
+
+LOGGER = get_task_logger(__name__)
+ACE_ROUTING_KEY = getattr(settings, 'ACE_ROUTING_KEY')
+
+
+@task(routing_key=ACE_ROUTING_KEY)
+def send_bulk_mail_to_newly_created_students(students):
+    """
+    A celery task, responsible to send registration email to newly created users.
+
+    Arguments:
+        students (list): A list of dicts containing information about newly created/updated
+                         users
+    """
+    new_students = [student for student in students if student.get('status') == 'Created']
```

Comment:
can we filter list before sending to this method, this should only responsible for sending email.

Reply:
done.